source/write/transactions.txt

```diff
@@ -134,7 +134,11 @@ following steps:
 
 If you require more control over your transactions, you can use the ``start_transaction()``
 method. You can use this method with the ``commit_transaction()`` and ``abort_transaction()``
-methods described in the preceding section to manually manage the transaction lifecycle. 
+methods described in the preceding section to manually manage the transaction lifecycle.
+
+.. note:: Parallel Operations Not Supported
+
+   {+driver-short+} does not support running parallel operations within a single transaction.
```

Comment:
Q: does PyMongo not support bulk operations on multiple namespaces? If so, I'd add that to the admonition and link to the Bulk Write guide

Reply:
Versions 4.9+ do, but we haven't documented it yet, so there's nothing to link to. (Could link to the [old docs domain](https://pymongo.readthedocs.io/en/stable/examples/client_bulk.html), but that seems like it could be confusing for people...?)

Reply:
Ah okay! then I think not linking is fine